src/Indicator.vala

```diff
@@ -541,6 +545,16 @@ public class Sound.Indicator : Wingpanel.Indicator {
 
         return true;
     }
+
+    private void update_tooltip () {
```

Comment:
This gets called very rapidly when operating the slider - even for tiny changes that do not change the tooltip.  Not sure how much this matters, but it is not really worth updating the tooltip while the popover is open as they cannot both be displayed.

Reply:
I _could_ use a `prev_value` and compare it to `current_value` and if it diffs by some `threshold` I'll update the tooltip.

Reply:
I can remember a time when processors were several orders of magnitude slower so maybe I am too concerned with efficiency.  Its probably not worth the extra code unless someone reports a noticreable lag.